src/styles/Home.module.css

```diff
@@ -1,278 +1,85 @@
-.main {
+.container {
+  background-color: #f6f5ff;
+}
+
+/* NON SESSION USER HOMEPAGE */
+.greetwrap {
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
+  justify-content: center;
   align-items: center;
-  padding: 6rem;
   min-height: 100vh;
 }
 
-.description {
-  display: inherit;
-  justify-content: inherit;
-  align-items: inherit;
-  font-size: 0.85rem;
-  max-width: var(--max-width);
-  width: 100%;
-  z-index: 2;
-  font-family: var(--font-mono);
-}
-
-.description a {
+.greet {
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
-  gap: 0.5rem;
-}
-
-.description p {
-  position: relative;
-  margin: 0;
-  padding: 1rem;
-  background-color: rgba(var(--callout-rgb), 0.5);
-  border: 1px solid rgba(var(--callout-border-rgb), 0.3);
-  border-radius: var(--border-radius);
-}
-
-.code {
-  font-weight: 700;
-  font-family: var(--font-mono);
-}
-
-.grid {
-  display: grid;
-  grid-template-columns: repeat(4, minmax(25%, auto));
-  width: var(--max-width);
-  max-width: 100%;
+  gap: 15px;
 }
 
-.card {
-  padding: 1rem 1.2rem;
-  border-radius: var(--border-radius);
-  background: rgba(var(--card-rgb), 0);
-  border: 1px solid rgba(var(--card-border-rgb), 0);
-  transition: background 200ms, border 200ms;
+/* SIGN IN BUTTON */
+.signin {
+  height: 40px;
+  padding: 0px 24px;
+  background-color: #5347d7;
+  border-radius: 5px;
+  border: none;
+  color: white;
+  cursor: pointer;
 }
 
-.card span {
-  display: inline-block;
-  transition: transform 200ms;
+.signin:hover {
+  box-shadow: 1px 1px 2px gray;
 }
 
-.card h2 {
-  font-weight: 600;
-  margin-bottom: 0.7rem;
+.signin:active {
+  background-color: #3023b8;
 }
 
-.card p {
-  margin: 0;
-  opacity: 0.6;
-  font-size: 0.9rem;
-  line-height: 1.5;
-  max-width: 30ch;
-}
-
-.center {
+/* SESSION USER PAGE */
+.userinfowrap {
```

Comment:
minor: Use hyphens to separate multiple words in a class name [1]. Here and other classes


[1] https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
For this, I initially tried adding a hyphen between all classname to separate multiple words, but when I do that - it gave me an error. I just did some research and found: 

- In TypeScript, hyphens are not allowed in class names because they are not valid in JavaScript syntax. If you're trying to use a hyphen in your class name, you should replace it with another character such as an underscore.

Just added an underscore for all multiple words class names~Pushing it to branch on the most recent commit.

Reply:
I think hyphenated css class names need to be quoted in a tsx file, e.g. 
 <div className={"styles.user-info-wrap"}>

Between underscores and hyphens: for css, hyphens are the default. In general, I strongly encourage to use the native style of the underlying languages, e.g. hyphenated-names in css, snake_case in C/C++, camelCase in Java etc.

Reply:
Gotchu! That makes sense, I'll try it out tomorrow along w updating the logo + fonts. 

Thank you Anup!